bridge/github/import.go

```diff
@@ -670,7 +475,7 @@ func (gi *githubImporter) getGhost(repo *cache.RepoCache) (*cache.IdentityCache,
 		"login": githubv4.String("ghost"),
 	}
 
-	err = gi.client.Query(context.TODO(), &q, variables)
+	err = gi.iterator.gc.Query(context.TODO(), &q, variables)
```

Comment:
it works, but it does tangle the importer and the iterator together. It would be better if the importer has it's own client. As it's a one shot call, you can probably create it just for this function.